### PR TITLE
Purge caption

### PR DESCRIPTION
**Pull request by RadekVana**
BFW-829

### CODE REVIEW

File: src/marlin_stubs/pause/M701_M702.cpp [hunk 1]
```diff
@@ -40,17 +40,18 @@
...
 #include "pause_stubbed.hpp"
 #include "filament.h"
+#include <functional>
 
 #define DO_NOT_RESTORE_Z_AXIS
 #define Z_AXIS_LOAD_POS   40
 #define Z_AXIS_UNLOAD_POS 20
 
-typedef void (*load_unload_fnc)(const int8_t target_extruder);
+using Func = std::function<void()>;
```
Comment (DRracer) [inline src/marlin_stubs/pause/M701_M702.cpp:49]: Are you sure std function doesn't allocate memory? Not sure about current state, but there were times when std::function allocated memory to store the pointer. Please check the template definition thoroughly

File: src/marlin_stubs/pause/pause.cpp [hunk 2]
```diff
@@ -97,13 +94,28 @@ Pause &Pause::GetInstance() {
     return s;
 }
 
-//xyze_pos_t Pause::resume_position;
+void Pause::SetUnloadLenght(float len) {
+    unload_length = -ABS(len); // it is negative value
```
Comment (DRracer) [inline src/marlin_stubs/pause/pause.cpp:98]: use std functions in c++ (std::abs), no need to use the c-style 3rd party workarounds

File: src/marlin_stubs/pause/M701_M702.cpp (hunk 1 above)
Comment (RadekVana) [reply src/marlin_stubs/pause/M701_M702.cpp:50]: it is fine, it should pass pointer to anonymous functor (lambda)

File: src/marlin_stubs/pause/pause.cpp (hunk 2 above)
Comment (RadekVana) [reply src/marlin_stubs/pause/pause.cpp]: thx
fixed

**Pull request merged**